examples/sdk-interactive-examples/examples-template/src/backend.ts

```diff
@@ -0,0 +1,12 @@
+// (C) 2022-2023 GoodData Corporation
+import { IAnalyticalBackend } from "@gooddata/sdk-backend-spi";
+import { withCaching, RecommendedCachingConfiguration } from "@gooddata/sdk-backend-base";
+import backendFactory, { ContextDeferredAuthProvider } from "@gooddata/sdk-backend-tiger";
+
+// Configure backend connection with context deferred auth provider.
```

Comment:
Do we need this comment for examples? (probably does not make sense in this context) or what do you think?